JumboCode/english-at-large
# Knumata5127/eng 18 crud operations books

# Description

Created the CRUD operations for books

## Issues






CRUD Operations for Books

## Screenshots

<img src="https://github.com/user-attachments/assets/b4d76c5d-336c-4085-b516-1735a7f3a52c" width=500>


## Test

- Open the Backend of the website

Create Book
- Click on "Create Book"
- If you open up Neon, you will see a new book created

Update Book
- Edit a book within the code base and assign it the ID of the book you want to update
- Put that book into the update function
- Click on "Update Book"
- If you open up Neon, you should see the book information change

Delete Book
- Edit a book within the code to match a book within the database
- Put the book into the delete function
- Click the "Delete Book" button on the website
- If you open up Neon, you will see the book disappear in the database.

GetOne Book
- Put the ID of the book you want into the getOne function (in the code)
- Click the "getOne Book" button on the website
- If you open up the terminal, there should be a message with the information of the book

GetAllBook
- Click the "getAll Book" button
- If you open up the terminal, there should be a number indicating how many books there are

## Possible Downsides or 

We haven't created a user input yet on the website so the user can't adjust change, delete, or getOne feature.

## Additional Documentations
